sensors.h

```diff
@@ -91,6 +94,11 @@ class AnalogSensor
     int getValue();
 }; 
 
+struct sensorData_t {
+    bool doorOpen; // door open?
+    bool motion; // motion?
```

Comment:
I liked what you had been doing when you were naming `bool` values starting with the word `is`. So `isDoorOpen` and `isMotion` (or `isThereMotion` to be consistent with the function name) to me was really nice.